docs/source/accelerators/gpu.rst

```diff
@@ -278,6 +278,7 @@ Lightning allows multiple ways of training
 - DistributedDataParallel 2 (``strategy='ddp2'``) (DP in a machine, DDP across machines).
 - Horovod (``strategy='horovod'``) (multi-machine, multi-gpu, configured at runtime)
 - TPUs (``tpu_cores=8|x``) (tpu or TPU pod)
+- Bagua (``strategy='bagua'``) (multiple-gpus across many machines (python script based))
```

Comment:
```suggestion
- [Bagua](https://github.com/BaguaSys/bagua) (``strategy='bagua'``) (multiple-gpus across many machines)
- TPUs (``tpu_cores=8|x``) (tpu or TPU pod)
```

I'd swap this around just so that we group the GPU DDP components together! I'd also consider adding a different description to explain why Bagua is different to say Horovod or DDP (probably something our throughput and it being advanced?)

Reply:
Would Bagua (strategy='bagua') (multiple-gpus across many machines with advanced training algorithms) be OK?